apps/events-helsinki/src/common-events/components/shareLinks/ShareLinks.tsx

```diff
@@ -13,7 +14,8 @@ export interface ShareLinksProps {
 }
 
 const ShareLinks: React.FC<ShareLinksProps> = ({ title }) => {
-  const { t } = useConfig();
+  const { t } = useTranslation(['common']);
```

Comment:
useCommonTranslation() is preferred, because it should have a better typing support.